src/App/components/FactsLoader/FactsLoader.js

```diff
@@ -1,28 +1,28 @@
-export async function FactsLoader (factQuantity, factId) {
-  const baseUrl = new URL('https://cat-fact.herokuapp.com/facts/591f98803b90f7150a19c229')
-  let newUrl
+export async function FactsLoader(factQuantity, factId) {
+  const baseUrl = new URL('https://cat-fact.herokuapp.com/facts/591f98803b90f7150a19c229');
+  let newUrl;
   if (!factId) {
-    newUrl = new URL('random', baseUrl)
-    newUrl.searchParams.set('animal_type', 'cat')
-    newUrl.searchParams.set('amount', factQuantity)
+    newUrl = new URL('random', baseUrl);
+    newUrl.searchParams.set('animal_type', 'cat');
+    newUrl.searchParams.set('amount', factQuantity);
   } else {
-    newUrl = new URL(factId, baseUrl)
+    newUrl = new URL(factId, baseUrl);
   }
 
   try {
     const result = await fetch(newUrl, {
       method: 'GET',
       headers: {
-        'Content-Type': 'application/json;charset=utf-8'
-      }
-    })
-    const resultJSON = await result.json()
+        'Content-Type': 'application/json;charset=utf-8',
+      },
+    });
+    const resultJSON = await result.json();
     if (!Array.isArray(resultJSON)) {
-      return [resultJSON]
+      return [resultJSON];
     } else {
-      return resultJSON
+      return resultJSON;
     }
   } catch (error) {
-    console.log(error)
+    console.log(error);
```

Comment:
if you added `no-console` eslint rule - why is it here?

this should be eliminated once we have husky